docs/Creator-V2-Whats-New.md

```diff
@@ -1,309 +1,195 @@
-The following examples give practical illustrations of how to use a popular [React Select](https://react-select.com/home) component: [React-Select in SurveyJS Library](https://surveyjs.io/Examples/Library?id=custom-widget-react-select) and [React-Select in SurveyJS Creator](https://surveyjs.io/Examples/Survey-Creator?id=react-select). 
-
-
-### Not a black box anymore
-With V2, SurveyJS Creator for React is not anymore a black box for React developers. You can now override Creator elements and change their render and behavior. Refer to the [Override the Property Grid's React component](https://surveyjs.io/Examples/Survey-Creator?id=overridepropertygrid&platform=ReactjsV2) example to see how to add a new element on top of the Creator's Property Grid.
+## Native Survey Creator for React
 
+New Survey Creator for React is built from true React components. The native implementation became possible because we separated the platform-independent model code (now distributed as the [`survey-creator-core`](https://github.com/surveyjs/survey-creator/tree/master/packages/survey-creator-core) package) from the platform-specific rendering code (the [`survey-creator-react`](https://github.com/surveyjs/survey-creator/tree/master/packages/survey-creator-react) package). You can override the default rendering functions of our React components and integrate your own React components (view the [Override the Property Grid Component](https://surveyjs.io/Examples/Survey-Creator?id=overridepropertygrid&platform=ReactjsV2) example). We implemented the same concept in the SurveyJS Library earlier.
 
+Most of the UI elements in the new Survey Creator (Property Grid, Translation and Logic Tabs, modal editors) are stylized surveys from our own SurveyJS Library for React. This feature allows you to integrate any 3rd-party React component into both Survey Creator and SurveyJS library. View examples that show how to integrate the [React Select](https://react-select.com/home) component [into a standalone survey](https://surveyjs.io/Examples/Library?id=custom-widget-react-select) or [into the Survey Creator](https://surveyjs.io/Examples/Survey-Creator?id=react-select).
 
 <a id="ui-changes"></a>
 
-## Major changes in UX/UI
-We have introduced a lot of changes in Creator V2. Actually, we have rewritten almost all code from scratch to make Creator's UI cleaner, more intuitive and adaptive.
-
-
-<a id="page-navigation"></a>
-
-### Page navigation improved
-We have completely changed the way a user works with surveys that have multiple pages. Instead of displaying pages in horizontally oriented tabs, we now use a Microsoft Word inspired page layout (all pages, one by one) with a vertically oriented page navigator widget.
+## Major UI/UX changes
 
-![Page navigator](images/creator-v2-page-navigator.png)  
-_Page navigator_
+We redesigned most parts of the Survey Creator to make its UI more adaptive and UX more intuitive. Refer to help topics in this section for details.
```

Comment:
Refer to help topics? Maybe "refer to new feature descriptions in this section for details"?